source/api/commands/as.md

```diff
@@ -46,22 +45,44 @@ The name of the alias to be referenced later within a {% url `cy.get()` get %} o
 
 ## DOM Element
 
+Aliasing a DOM element and then {% url getting get %} that aliased element.
```

Comment:
Style guide steer Jennifer: is linking to commands using phrases or plain words such as _"getting"_ for `cy.get()` cool? The sentence reads smoother in this style but I can't recall seeing an example of this style elsewhere in the docs.

Reply:
No, we have not had a use case of this before. We generally write `cy.get()` here. I would probably use `cy.get()` for more clarity - this explains the command needed without them needed to click on the actual link to see the example.